group_project/group_project.py

```diff
@@ -136,15 +136,31 @@ def milestone_dates(self):
         group_activity = GroupActivity.import_xml_string(self.data, self.is_admin_grader)
         return group_activity.milestone_dates
 
+    def _get_project_api_location(self):
+        location = '.project_api.ProjectAPI'
+        if hasattr(settings, 'GROUP_PROJECT_API_LOCATION'):
+            location = settings.GROUP_PROJECT_API_LOCATION
+
+        splitted = location.split('.')
+        return ".".join(splitted[:-1]), splitted[-1], not splitted[0]
```

Comment:
I think this would also be nicer if you do not allow relative imports. Just make the default `group_project.project_api.ProjectAPI`, disable support for group imports, and then you can use

```python
module, class_name = location.rsplit('.', 1)
return getattr(importlib.import_module(module), class_name)
```

P.S. FYI, the past tense of "split" is just "split" - so "splitted" is wrong.